examples/jsm/nodes/accessors/InstanceNode.js

```diff
@@ -26,7 +25,7 @@ class InstanceNode extends Node {
 
 		const instanceBufferNode = buffer( instanceMesh.instanceMatrix.array, 'mat4', instanceMesh.count );
 
-		this.instanceMatrixNode = temp( element( instanceBufferNode, instanceIndex ) );
+		this.instanceMatrixNode = element( instanceBufferNode, instanceIndex );
```

Comment:
This cache was on purpose, did you check the WGSL code?

Reply:
No, I did not... Then it can be replaced by `label`.

Reply:
I think that `temp` is a name more intuitive to force a variable creation, and the name can be optional.

Another usage for `temp( node, propertyName = null )` can be: 

```js
const vec = temp( vec3() );

vec.add( ...someNode );
```

Reply:
Fixed.